jest-preset.json

```diff
@@ -1,9 +1,9 @@
 {
   "globals": {
     "ts-jest": {
-      "tsConfigFile": "src/tsconfig.spec.json"
-    },
-    "__TRANSFORM_HTML__": true
+      "tsConfig": "src/tsconfig.spec.json",
+       "stringifyContentPathRegex": true
+    }
```

Comment:
we should also bump our dependency of ts-jest to be newer, right? as I assume this is breaking

Reply:
Indeed it is more like breaking change. I think the whole changes should include changing jest-preset, bumping ts-jest, adjust preprocessor. I'm not sure if changing jest-preset is enough.

Reply:
right. sorry will do some more work to get it done right